GruntFile.js

```diff
@@ -125,6 +128,18 @@ module.exports = function(grunt) {
                     ]
                 }
             }
+        },
+        exec: {
+            "meteor-init": {
+                // Make sure that Meteor is installed.
+                command: "type meteor > /dev/null 2>&1 || { curl https://install.meteor.com/ | sh; }"
```

Comment:
So I'll admit that I was just somewhat blindly following the guide. If you think that Gulp is a better method, then I say let's go for it. I personally don't think that Meteor is developed enough in terms of workflows and continuous integration to dictate the "proper" way to setup Grunt, Travis, etc. I have never used Gulp. Do you see a superior solution? If so, I am all for it.

-----Original Message-----
From: "liquidpele" <notifications@github.com>
Sent: ‎9/‎25/‎2015 10:45 PM
To: "novus/nvd3" <nvd3@noreply.github.com>
Cc: "Joel" <joel.lubrano@gmail.com>
Subject: Re: [nvd3] Integrate Meteor Tasks Into GruntFile (#1260)

In GruntFile.js:
> @@ -125,6 +128,18 @@ module.exports = function(grunt) {
>                      ]
>                  }
>              }
> +        },
> +        exec: {
> +            "meteor-init": {
> +                // Make sure that Meteor is installed.
> +                command: "type meteor > /dev/null 2>&1 || { curl https://install.meteor.com/ | sh; }"
oh, it literally says to do just that on that page. wow. that's terrible. 
—
Reply to this email directly or view it on GitHub.